frontend/src/components/Navbar.tsx

```
@@ -19,66 +20,184 @@ const roboto = Roboto({
+              </button>
+            </div>
+          </div>
+        </div>
+      </nav>
+
+      {/* Mobile/Tablet Menu Overlay */}
+      {isMobileMenuOpen && (
+        <div
+          className="fixed inset-0 bg-black bg-opacity-50 z-30 lg:hidden"
+          onClick={closeMobileMenu}
+        />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)